### PR TITLE
Feature loginsignup integration

### LINKED ISSUE #76: Check login/signup inputs for validity, give user feedback if not
The inputs given to the login and signup textfields should be checked. We need to make sure that all the inputs have been provided. If an input isn't provided, we should notify the user that it was missing.
Inputs should also be processed to some degree. At a minimum, we should remove leading/trailing whitespace

Depends on #10 & #11

### PR DESCRIPTION
**Pull request by max-gurr**
- Added input type to textInput component, so we can specify field types, e.g. password or email. "text" is used when a type is not given
- Check login/signup inputs to make sure all values are provided. If an input is missing, placeholder text is used to let the user know
- If all inputs are there, redirect to the relevant page
  - From signup -> login
  - From login -> home
- Also some classname changes for consistency and readability

NOTE: Firebase methods will be integrated in a coming PR. These will be used to check a valid signup/login before redirecting

### CODE REVIEW

File: src/pages/loginView.js [hunk 1]
```diff
@@ -1,41 +1,68 @@
...
+            placeholderValue={passwordInfoText}
             onChangeHandler={setPassword}
             type='password'
           />
         </div>
-        <div onClick={login}>
-          <Button icon={'rightArrow'} text={'Login'} />
+        <Button icon='rightArrow' text='Login' handleOnClick={login} />
+        <div className="login__bottom-text">
+          <span>Don&apos;t have an account?</span>
```
Comment (franc-hoon) [inline src/pages/loginView.js:62]: Just curious why a normal ' doesn't work here?

**Review by franc-hoon — APPROVED** [review]
LGTM 👍

File: src/pages/loginView.js [hunk 2]
```diff
@@ -1,41 +1,68 @@
...
-    console.log(email, password);
+  const [ emailInfoText, setEmailInfoText ] = useState("");
+  const [ passwordInfoText, setPasswordInfoText ] = useState("");
+  
+  const login = () => {
+    // Process inputs
+    const processedEmail = email.trim();
+    const processedPassword = password.trim();
+    
+    //
```
Comment (yuno99825) [inline src/pages/loginView.js]: Empty comment here

**Review by yuno99825 — APPROVED** [review]
Looks good, great job

File: src/pages/loginView.js (hunk 1 above)
Comment (max-gurr) [reply src/pages/loginView.js:62]: this was a linting suggestion, I think's for browser/device compatability

**Pull request merged**